blog/_posts/2020-09-09-github-issues.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Issues? GitHub!"
+layout: archive
+author: Axel Naumann
+---
+
+First of all: a big THANK YOU for submitting bugs for ROOT.
+We appreciate the effort: finding back the issue submission web page, explaining what the problem is, and sometimes even providing a reproducer.
+
+To make it easier for you to submit issues, we now switched from ~~Jira~~ to [GitHub Issues](https://github.com/root-project/root/issues/new).
+We hope that having a GitHub account is so common these days that you're already logged in and used to GitHub's interfaces.
+
+O, and Markdown instead of `{code}`. Phew!
+
+Starting today, new bugs will only be reported in GitHub.
+We will keep fixing the existing issues in Jira until they are all resolved.
```

Comment:
It would be good to clarify that we are not migrating existing jira tickets to github

Reply:
Thanks, addressed, please check.